content/docs/dvclive/get-started.md

```diff
@@ -22,6 +22,51 @@ live = Live()
 
 See [`Live()`](/doc/dvclive/api-reference/live) for details.
 
+### Log parameters
```

Comment:
```suggestion
### (Optionally) Log parameters
```
Technically, it's all optional 😄 , but it might be worth making explicit that you don't have to log params.